worker/routes/practiceAssistant.ts

```
@@ -0,0 +1,124 @@
+}
+
+export function runPracticeAssistantTurn(params: PracticeAssistantTurnParams): Response {
+  const { conversationId, practiceId, practiceSlug, userMessage, userId, auth, env, request, messages } = params;
+  const { response, write, close } = createSseResponse();
+  Logger.info('practice_assistant.sse.prepared', {
+    conversationId,
+    practiceId,
+    messageCount: messages?.length ?? null,
+    userMessageLength: userMessage.length,
+  });
+  queueMicrotask(async () => {
```

ESLint

[error] 'queueMicrotask' is not defined.

(no-undef)